root/report/PseudoReleasesWithCoverArt.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * @flow strict
+ * Copyright (C) 2025 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import ReleaseList from './components/ReleaseList.js';
+import ReportLayout from './components/ReportLayout.js';
+import type {ReportDataT, ReportReleaseT} from './types.js';
+
+component PseudoReleasesWithCoverArt(...{
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportReleaseT>) {
+  return (
+    <ReportLayout
+      canBeFiltered={canBeFiltered}
+      description={exp.l(
+        `This report shows pseudo-releases that have cover art in the Cover
```

Comment:
In root/report/EventsWithEaaNoTypes.js we used "images in the Event Art Archive", which sounds less repetitive to me:
```suggestion
        `This report shows pseudo-releases that have images in the Cover
```